README.md

```diff
@@ -192,6 +192,13 @@ python prompt.py -e {experiment_name}
 where `experiment_name` is the output folder of the experiment you want to chat with (see configuration).
 The interactive chat will also work with model that were finetuned using the UI.
 
+To publish the model to Hugging Face, use the following command:
+
+```bash
+python publish_to_hugging_face.py -e {experiment_dir} -d {device} -a {api_key} -r {repository_id}

```

Comment:
let's also add an option for the target name on HF

and is it possible to not need to login? if one is already logged in in the OS?

Reply:
Do you mean something like this?

# Define the path for the model files and upload them to the target repository
    model_path = os.path.join(experiment_path, "model")
    repo_url = f"{user_id}/{target_name}"
    repo = huggingface_hub.HfApi().create_repo(token=api_key, name=target_name)
    huggingface_hub.HfApi().upload(repo=repo, path=model_path)